### PR TITLE
Added two bullet points about Convertible RI's and Regional Benefit

### PR DESCRIPTION
**Pull request by billjive**
AWS just released a new RI class (Convertible) and a new RI attribute (Regional Benefit). I added a short statement about how this class is different and when you'd want to use them. Same goes for Regional Benefit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1355,6 +1355,9 @@ Billing and Cost Management
...
 	-	Instance reservations are not tied to specific EC2 instances - they are applied at the billing level to eligible compute hours as they are consumed across all of the instances in an account.
+	-	[Convertible Reserved Instances](https://aws.amazon.com/blogs/aws/ec2-reserved-instance-update-convertible-ris-and-regional-benefit/) are a new form of Reserved Instance that allows changes during the term. Convertible RI's are more flexible than Standard RI's in exchange for a smaller savings rate. Use these if you can make a long-term commitment (3 years) but want the flexiblity to move to new architectures or take advantage of future price discounts.
+	-	Consider setting the "Regional Benefit" attribute of an RI if you don't need a capacity guarantee. The upside of this is your RI will potentially be used more, especially if you have a lot of compute inside of one Region.
+	Instance reservations are not tied to specific EC2 instances - they are applied at the billing level to eligible compute hours as they are consumed across all of the instances in an account.
```
Comment (jlevy) [inline README.md]: Is there a link on this?

Comment (jlevy) [inline README.md]: Nit: use em dash: —

File: README.md [hunk 2]
```diff
@@ -1355,6 +1355,9 @@ Billing and Cost Management
...
 	-	Instance reservations are not tied to specific EC2 instances - they are applied at the billing level to eligible compute hours as they are consumed across all of the instances in an account.
+	-	[Convertible Reserved Instances](https://aws.amazon.com/blogs/aws/ec2-reserved-instance-update-convertible-ris-and-regional-benefit/) are a new form of Reserved Instance that allows changes during the term. Convertible RI's are more flexible than Standard RI's in exchange for a smaller savings rate. Use these if you can make a long-term commitment (3 years) but want the flexiblity to move to new architectures or take advantage of future price discounts.
+	-	Consider setting the "Regional Benefit" attribute of an RI if you don't need a capacity guarantee. The upside of this is your RI will potentially be used more, especially if you have a lot of compute inside of one Region.
```
Comment (jlevy) [inline README.md]: Nit: Use “Unicode quotes” and Unicode’s apostrophes

File: README.md [hunk 3]
```diff
@@ -1355,6 +1355,9 @@ Billing and Cost Management
...
 	-	Consider using Reserved Instances when you can predict your longer-term compute needs and need a stronger guarantee of compute availability and continuity than the (typically cheaper) Spot market can provide. However be aware that if your architecture changes your computing needs may change as well so long term contracts can seem attractive but may turn out to be cumbersome.
 	-	Instance reservations are not tied to specific EC2 instances - they are applied at the billing level to eligible compute hours as they are consumed across all of the instances in an account.
+	-	[Convertible Reserved Instances](https://aws.amazon.com/blogs/aws/ec2-reserved-instance-update-convertible-ris-and-regional-benefit/) are a new form of Reserved Instance that allows changes during the term. Convertible RI's are more flexible than Standard RI's in exchange for a smaller savings rate. Use these if you can make a long-term commitment (3 years) but want the flexiblity to move to new architectures or take advantage of future price discounts.
```
Comment (jlevy) [inline README.md]: RI's -> RIs (two places)

Comment (jlevy) [inline README.md]: Optional: Are there any good non-AWS blogs or info we should link to wrt this with more analysis?

File: README.md (hunk 1 above)
Comment (billjive) [inline README.md]: Best link I've seen that's not from a vendor: http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/concepts-reserved-instances-application.html

File: README.md (hunk 3 above)
Comment (billjive) [inline README.md]: Yes, but they're all from vendors (including myself!)

**Comment by billjive** [comment]
Updated with all my changes...

File: README.md (hunk 1 above)
Comment (jlevy) [inline README.md]: Sure, want to add that and fix the dash?

> Instance reservations are not tied to specific EC2 instances — they are [applied at the billing level](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/concepts-reserved-instances-application.html) to eligible compute hours as they are consumed across all of the instances in an account.

**Comment by jlevy** [comment]
Looks good from my side with the one small addition. Still would like @max-zanko or @ThanosBaskous to give it a look. Thanks!

File: README.md [hunk 4]
```diff
@@ -1354,7 +1354,10 @@ Billing and Cost Management
...
-	-	Instance reservations are not tied to specific EC2 instances - they are applied at the billing level to eligible compute hours as they are consumed across all of the instances in an account.
+	-	Instance reservations are not tied to specific EC2 instances — they are applied at the [billing level to eligible compute hours as they are consumed across all of the instances in an account](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/concepts-reserved-instances-application.html).
+	-	[Convertible Reserved Instances](https://aws.amazon.com/blogs/aws/ec2-reserved-instance-update-convertible-ris-and-regional-benefit/) are a new form of Reserved Instance that allows changes during the term. Convertible RIs are more flexible than Standard RIs in exchange for a smaller savings rate. Use these if you can make a long-term commitment (3 years) but want the flexiblity to move to new architectures or take advantage of future price discounts.
+	-	Consider setting the “Regional Benefit” attribute of an RI if you don't need a capacity guarantee. The upside of this is your RI will potentially be used more, especially if you have a lot of compute inside of one Region.
```
Comment (ThanosBaskous) [inline README.md]: Nit: the standard in the guide is unicode apostrophes - "don't" -> "don‘t".

Comment (ThanosBaskous) [inline README.md]: Phrasing suggestion: "The upside of this is your RI will..." -> ""The upside of this is that your RI will...""

Comment (ThanosBaskous) [inline README.md]: Consistency: don't capitalize region.

File: README.md [hunk 5]
```diff
@@ -1354,7 +1354,10 @@ Billing and Cost Management
...
+	Instance reservations are not tied to specific EC2 instances - they are applied at the billing level to eligible compute hours as they are consumed across all of the instances in an account.
```
Comment (ThanosBaskous) [inline README.md]: Bad merge / accidental duplicate? I think this is point is present, in updated form, on line 1357 above.

File: README.md (hunk 4 above)
Comment (min-polyvore) [inline README.md]: I agree to change it, "only want the discount" is a pretty big use case in lots of small start-ups.

**Comment by jlevy** [comment]
@billjive do you think you can update this so we can push it through? Thanks again!

**Comment by QuinnyPig** [comment]
@billjive has gone quiet; closing pending his reappearance!

**Pull request closed without merging**